libs/langchain/langchain/agents/openai_functions_agent/base.py

```diff
@@ -226,3 +227,120 @@ def from_llm_and_tools(
             callback_manager=callback_manager,
             **kwargs,
         )
+
+
+def create_default_prompt(
+    system_message: Optional[str] = "You are a helpful AI assistant.",
+    include_chat_history: bool = False,
+):
+    """Create default prompt for OpenAI Functions agent.
+
+    Args:
+        system_message: String to use as the system message,
+            defaults to "You are a helpful AI assistant."
+        include_chat_history: Whether to include a variable for chat history or not.
+            If yes, this will require an input of `chat_history`.
+
+    Returns:
+        A prompt template. Requires two or three variables:
+        - `input`: this is the user input
+        - `agent_scratchpad`: this is the history of steps the agent has taken so far.
+            This gets populated by the AgentExecutor.
+        - `chat_history`: This is optionally required as input, if
+            `include_chat_history` is set to True. It should be a list of LangChain
+            Message objects.
+    """
+    messages: List[Union[BaseMessagePromptTemplate, BaseMessage]]
+    messages = [SystemMessage(content=system_message)]
+    if include_chat_history:
+        messages.append(MessagesPlaceholder(variable_name="chat_history"))
```

Comment:
I would really love to find a good way to standardize or reserve this name - always struck me as a bit arbitrary and inconsistent across modules

Reply:
Possibly as memory becomes more defined